### PR TITLE
improved_launcher

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

**Review by fievelk — APPROVED** [review]

**Review by Hassan754 — DISMISSED** [review]

**Review by faizan2590 — DISMISSED** [review]

**Review by spyrostz — DISMISSED** [review]
LGTM

File: src/d3a/d3a_core/launcher.py [hunk 1]
```diff
@@ -35,7 +36,7 @@ class Launcher:
     def __init__(self,
                  queue=None,
                  max_jobs=None,
-                 max_delay_seconds=2):
+                 max_delay_seconds=5):
```
Comment (hannesdiedrich) [inline src/d3a/d3a_core/launcher.py:39]: please reduce to 2 again. thanks!

Comment (spyrostz) [reply src/d3a/d3a_core/launcher.py]: Done, thanks!

**Review by mclrch — APPROVED** [review]
LGTM

**Pull request merged**